pyanaconda/kickstart.py

```diff
@@ -1814,6 +1814,13 @@ def __init__(self, *args, **kwargs):
 
         commands.repo.F27_RepoData.__init__(self, *args, **kwargs)
 
+    def __str__(self):
+        """Don't output disabled repos"""
+        if self.enabled:
+            return super(RepoData, self).__str__(self)
```

Comment:
I'm getting exception thanks to this line. Please change ti to `super().__str__()`.